test/eth.accounts.sign.js

```diff
@@ -31,15 +31,15 @@ describe("eth", function () {
     describe("accounts", function () {
 
         tests.forEach(function (test, i) {
-            it("sign data using a string", function() {
+            it("sign data using a string", function () {
```

Comment:
```suggestion
            it("sign data using a string", function() {
```

Reverting this would preserve the stylistic consistency of this file / minimize the diff. Is the space between function and () automatically added by your editor? If so, perhaps that could be disabled? 🙂

Reply:
Good point 😅 I will configure my editor correctly asap.